--- src/AnalyticsClientFactory.php      2024-04-28 15:09:20.958817807 -0700
+++ src/AnalyticsClientFactory.php      2024-04-28 15:10:17.240889263 -0700
@@ -16,9 +16,9 @@

     public static function createAuthenticatedGoogleClient(array $config): BetaAnalyticsDataClient
     {
-        return new BetaAnalyticsDataClient([
+        return new BetaAnalyticsDataClient(!empty($config['service_account_credentials_json']) ? [
             'credentials' => $config['service_account_credentials_json'],
-        ]);
+        ] : []);
     }

     protected static function createAnalyticsClient(
--- src/AnalyticsServiceProvider.php    2024-04-28 15:06:42.275796910 -0700
+++ src/AnalyticsServiceProvider.php    2024-04-28 15:07:13.682394809 -0700
@@ -44,7 +44,7 @@
             return;
         }

-        if (! file_exists($analyticsConfig['service_account_credentials_json'])) {
+        if (!empty($analyticsConfig['service_account_credentials_json']) && ! file_exists($analyticsConfig['service_account_credentials_json'])) {
             throw InvalidConfiguration::credentialsJsonDoesNotExist($analyticsConfig['service_account_credentials_json']);
         }
     }
